torchgeo/datamodules/inria.py

```diff
@@ -0,0 +1,236 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""InriaAerialImageLabeling datamodule."""
+
+from typing import Any, Dict, List, Optional, Tuple, Union, cast
+
+import kornia.augmentation as K
+import pytorch_lightning as pl
+import torch
+import torchvision.transforms as T
+from einops import rearrange
+from kornia.contrib import compute_padding, extract_tensor_patches
+from torch.nn.modules.utils import _pair
```

Comment:
We should probably avoid using undocumented private methods of external libs. Can you use `..samplers.utils._to_tuple` instead?

Reply:
Done